openhtf/util/threads.py

```diff
@@ -60,7 +60,7 @@ def run(self):
       self._thread_proc()
     except Exception:  # pylint: disable=broad-except
       if not self._thread_exception(*sys.exc_info()):
-        _LOG.exception('Thread raised an exception: %s', self.name)
+        _LOG.critical('Thread raised an exception: %s', self.name)
```

Comment:
similar to the other exception -> critical, consider not giving up the stack trace.. when you want it there, you *really* want it there.

Reply:
Using log.exception doesn't make sense when we're about to re-raise on the next line, and as Arun mentions, ends up resulting in the same stack trace showing up multiple times. The documentation for logging.exception says it should only be used in exception handlers. Since we're re-raising we should leave it up to the configured exception handler whether/how to express the stack trace. For OpenHTF's usage we're overriding this to propagate exception info up via phase result as discussed in the other place this comes up.